pennylane/transforms/defer_measurements.py

```diff
@@ -776,16 +776,15 @@ def f():
             "must support the Projector gate to apply postselection."
         )
 
-    if len(reused_measurement_wires) > 0 and not all(isinstance(w, int) for w in tape.wires):
-        raise ValueError(
-            "qml.defer_measurements does not support custom wire labels with qubit reuse/reset."
-        )
+    integer_wires = list(filter(lambda w: isinstance(w, int), tape.wires))
```

Comment:
```suggestion
    integer_wires = [w for w in tape.wires if isinstance(w, int)]
```
Just a bit easier to read